SIPS/sip-12.md

```diff
@@ -44,7 +44,7 @@ As can be seen lately, congestion is the new norm in Ethereum, so the Dynamic op
 
 The proposal for Dynamic gas pricing is as follows:
 
-- To use a gas price oracle that gives us `fatest` (as soon as possible) gas price on-chain, which will update whenever the APIs providing these rates deviate more than some small percentage.
+- To use a gas price oracle that gives us `fast` (roughly < 2 min to confirm) gas price on-chain, which will update whenever the APIs providing these rates deviate more than some small percentage.
 - To update `Synthetix.sol`, removing the lock from exchanges and adding prevention of using more gas than the gas price oracle allows above;
```

Comment:
@justinjmoses  We will look at deprecating the lock logic from Synthetix.sol after the max gas price mechanism is stable, maybe we need to add this into SIP.